test/at_ex/gateway/Payments/Bank/transfer_test.ex

```diff
@@ -0,0 +1,134 @@
+defmodule AtEx.Gateway.Payments.Bank.TransferTest do
```

Comment:
Would it be possible to have a test for insufficient funds  as it is in the docs some thing like this 
`{
    "entries": [{
        "accountNumber": "1234567890",
        "status": "Queued",
        "transactionId": "ATPid_SampleTxnId",
        "transactionFee": "NGN 50.00"
    }, {
        "accountNumber": "1234567891",
        "status": "Failed",
        "errorMessage": "Insufficient Credit"
    }]
}`

Reply:
Never mind, I think we cant mock something like `"Insufficient Credit"` its all good.

Reply:
yeah, exactly